test/unit/platform.js

```diff
@@ -268,6 +268,24 @@ describe("platform", ()=>{
     });
   });
 
+  it("creates a folder recursively", ()=>{
+    mock(platform, "callMkdirp").callbackWith(null);
+
+    return platform.mkdirRecursively("folder1/folder2").then((err)=>{
+      assert(platform.callMkdirp.called);
```

Comment:
@fjvallarino could we use https://github.com/jupiter/simple-mock#spylastcall to assert that it's also called with `"folder1/folder1"` as the parameter?